cellpy/utils/easyplot.py

```diff
@@ -224,92 +205,129 @@ def verify_input(self):
+
+
 
     def fill_input(self):
-        # Fill in the rest of the variables from self.user_params if the user didn't specify
+        """Fill in the rest of the variables from self.user_params if the user didn't specify"""
         # Can't just join dicts since they have differing formats, need to loop...
         for key in self.user_params:
             try:
                 self.kwargs[key]
             except KeyError as e:
                 self.kwargs[key] = self.user_params[key][1]
 
-    def set_arbin_sql_credentials(
-        self, server="localhost", uid="sa", pwd="Changeme123", driver="ODBC Driver 17 for SQL Server"
-    ):
-        cellpy.prms.Instruments.Arbin["SQL_server"] = server
-        cellpy.prms.Instruments.Arbin["SQL_UID"] = uid
-        cellpy.prms.Instruments.Arbin["SQL_PWD"] = pwd
-        cellpy.prms.Instruments.Arbin["SQL_Driver"] = driver
-        self.use_arbin_sql = True
+
+
+    def set_arbin_sql_credentials(self, server = "localhost", uid="sa", pwd="Changeme123", driver="ODBC Driver 17 for SQL Server"):
+            """Sets cellpy.prms.Instruments.Arbin details to fit what is inserted.
+            Parameters: Server = 'IP of server', uid = 'username', pwd = 'password', driver = 'ODBC Driver 17 for SQL Server' """
+            cellpy.prms.Instruments.Arbin["SQL_server"] = server
+            cellpy.prms.Instruments.Arbin["SQL_UID"] = uid
+            cellpy.prms.Instruments.Arbin["SQL_PWD"] = pwd
+            cellpy.prms.Instruments.Arbin["SQL_Driver"] = driver
+            self.use_arbin_sql = True
+
+
 
     def give_color(self):
-        # Picks the first color from the color list and gives it away
+        """Picks the first color from the color list and gives it away"""
         color = self.colors[0]
-        self.colors = self.colors[1:]
+        self.colors=self.colors[1:]
         return color
 
+
+
     def give_fig(self):
+        """Gives figure to whoever asks and appends it to figure list"""
         fig, ax = plt.subplots(figsize=(6, 4))
         self.figs.append((fig, ax))
         return (fig, ax)
 
+
+
     def handle_outpath(self):
+        """Makes sure that self.outpath exists, or creates it."""
         if os.path.isdir(self.kwargs["outpath"]):
             return self.kwargs["outpath"]
         elif not os.path.isdir(self.kwargs["outpath"]):
             try:
                 os.makedirs(self.kwargs["outpath"])
                 return self.kwargs["outpath"]
             except OSError as e:
-                logging.error(
-                    "Cannot create output directory "
-                    + self.kwargs["outpath"]
-                    + ". Please make sure you have write permission. Errormessage: "
-                    + e
-                )
+                logging.error("Cannot create output directory " + self.kwargs["outpath"] + ". Please make sure you have write permission. Errormessage: " + e)
```

Comment:
A more "beautiful" way to write this is by using f-strings (it is also faster) .

`logging.error(f"Cannot create output directory {self.kwargs["outpath"]}. Please make sure you have write permission. Errormessage:  {e}")`